tests/signal/test_hampel_filter.py

```diff
@@ -0,0 +1,64 @@
+"""Test_hampel_filter.
+
+- Author: Sunjin.kim
+- Contact: sunjin.kim@onepredict.com
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+
+from onebone.signal import hampel_filter
+
+
+def timeseries_data(outlier: bool = True) -> np.ndarray:
+
+    """
+    make example timeseries
+    You can set the number of random outliers.
+
+    Parameters
+    ----------
+    num_outlier : bool
+        You can generate/degenerate timeseries data with example outliers.
+        default = True
```

Comment:
```suggestion
    num_outlier : bool, default=True
        You can generate/degenerate timeseries data with example outliers.
```
테스트라 document가 크게 중요하진 않지만 작성하신김에 format을 잘맞추는게 좋아보입니다.

Reply:
> 테스트라 document가 크게 중요하진 않지만 작성하신김에 format을 잘맞추는게 좋아보입니다.

format 을 맞추어, return 데이터에 대한 설명, example 도 추가하였습니다!